core/src/main/java/com/linecorp/armeria/common/util/Sampler.java

```diff
@@ -80,7 +53,45 @@ public boolean isSampled() {
      *
      * @param rate minimum sample rate is 0.01, or 1% of traces
      */
-    public static Sampler create(float rate) {
-        return CountingSampler.create(rate);
+    static <T> Sampler<T> random(double rate) {
+        @SuppressWarnings("unchecked")
+        final Sampler<T> cast = CountingSampler.create(rate);
+        return cast;
+    }
+
+    /**
+     * Returns a sampler, given a rate-limited on a per-second interval.
+     *
+     * @param samplesPerSecond minimum rate-limited is 0 and the max is 2,147,483,647 (max int)
+     */
+    static <T> Sampler<T> rateLimited(int samplesPerSecond) {
+        @SuppressWarnings("unchecked")
+        final Sampler<T> cast = RateLimitingSampler.create(samplesPerSecond);
+        return cast;
     }
+
+    /**
+     * Returns a sampler that always will be sampled.
+     */
+    static <T> Sampler<T> always() {
+        @SuppressWarnings("unchecked")
+        final Sampler<T> cast = Samplers.ALWAYS;
+        return cast;
+    }
+
+    /**
+     * Returns a sampler that never will be sampled.
+     */
+    static <T> Sampler<T> never() {
+        @SuppressWarnings("unchecked")
+        final Sampler<T> cast = Samplers.NEVER;
+        return cast;
+    }
+
+    /**
+     *  Returns true if a request should be recorded.
```

Comment:
- Double spaces between `*` and `Returns`.
- `{@code true}`

Reply:
Thank you for review. fiexed in 6459312.